.github/workflows/test-devcontainer.yaml

```
@@ -0,0 +1,65 @@
+name: Test DevContainer Image
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

```suggestion
on:
  push:
  pull_request:
```

We want the CI running for the other branches as well.